components/activity_and_insights/insights/top_dms_and_new_members/top_dms_and_new_members.test.tsx

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+import {Provider} from 'react-redux';
+
+import {act} from '@testing-library/react';
+
+import {ReactWrapper} from 'enzyme';
+
+import {BrowserRouter} from 'react-router-dom';
+
+import {CardSizes, InsightsWidgetTypes, TimeFrames} from '@mattermost/types/insights';
+
+import {mountWithIntl} from 'tests/helpers/intl-test-helper';
+import mockStore from 'tests/test_store';
+
+import TopDMsAndNewMembers from './top_dms_and_new_members';
+
+const actImmediate = (wrapper: ReactWrapper) =>
+    act(
+        () =>
+            new Promise<void>((resolve) => {
+                setImmediate(() => {
+                    wrapper.update();
+                    resolve();
+                });
+            }),
+    );
+
+jest.mock('mattermost-redux/actions/insights', () => ({
+    ...jest.requireActual('mattermost-redux/actions/insights'),
+    getMyTopDMs: () => ({type: 'adsf',
+        data: {
+            has_next: false,
+            items: [
+                {
+                    post_count: 17,
+                    second_participant: {
+                        id: 'q7bgtp9dtfdwxriz1e8p8zon3e',
+                        last_picture_update: 0,
+                        first_name: 'Aaron',
+                        last_name: 'Medina',
+                        nickname: 'Centidel',
+                        username: 'aaron.medina',
+                        position: 'Systems Administrator I',
+                    },
```

Comment:
Not sure if it's necessary, can we also add `outgoing_message_count` to this testdata?